source/connect.txt

```diff
@@ -235,9 +235,13 @@ Disconnect from a Deployment
 
 To disconnect from a deployment and exit ``mongosh``, you can:
 
-- Type ``.exit``, or
+- Type ``.exit``, ``exit``, or ``exit()``
```

Comment:
nit: Per the [style guide](https://docs.mongodb.com/meta/style-guide/style/lists/#list-items), end complete sentences in a list with a period. Should apply to all items in this list.